skunkworks_crow/src/main/java/org/odk/share/activities/InstancesList.java

```diff
@@ -32,21 +32,21 @@
 
 public class InstancesList extends InstanceListActivity implements LoaderManager.LoaderCallbacks<Cursor> {
 
-    @BindView(R.id.recyclerview) RecyclerView recyclerView;
-    @BindView(R.id.toolbar) Toolbar toolbar;
-    @BindView(R.id.send_button) Button sendButton;
-    @BindView(R.id.toggle_button) Button toggleButton;
-
-    private static final String INSTANCE_LIST_ACTIVITY_SORTING_ORDER = "instanceListActivitySortingOrder";
-
     public static final String INSTANCE_IDS = "instance_ids";
-
+    private static final String INSTANCE_LIST_ACTIVITY_SORTING_ORDER = "instanceListActivitySortingOrder";
```

Comment:
Why have you modified this file? There doesn't seem to be any fix related to code quality. Please revert the changes to this file if not required